src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/JaggaerService.java

```diff
@@ -119,52 +123,47 @@ public CreateUpdateRfxResponse createUpdateRfx(final Rfx rfx, final OperationCod
   public ExportRfxResponse getRfx(final String externalEventId) {
 
     final var exportRfxUri = jaggaerAPIConfig.getExportRfx().get(ENDPOINT);
-    return ofNullable(jaggaerWebClient.get().uri(exportRfxUri, externalEventId).retrieve()
-        .bodyToMono(ExportRfxResponse.class)
-        .block(ofSeconds(jaggaerAPIConfig.getTimeoutDuration())))
-            .orElseThrow(() -> new JaggaerApplicationException(INTERNAL_SERVER_ERROR.value(),
-                "Unexpected error retrieving rfx"));
+    log.info("Start calling Jaggaer API to get rfx, Rfx Id: {}", externalEventId);
```

Comment:
Have we tested what the effect of removing the thrown nulls here is?

Reply:
This code is extracted to a below method as this code is duplicated in several places. 
getExportRfxResponse(externalEventId, exportRfxUri);
And calling above method from line 127.